src/domain/user-group/user-group.entity.ts

```diff
@@ -84,6 +84,8 @@ export class UserGroup extends BaseEntity implements IUserGroup {
 }
 
 export enum RestrictedGroupNames {
-  Admins = 'admins',
+  Admins = 'ecoverse-admins',
```

Comment:
Should this not be also called "EcoverseAdmins" to be consistent?

Reply:
Fixed in d6f43ad5ea6b78b4b79d6889962971a5eae009eb